doc/aaa_session_metrics.md

```diff
@@ -0,0 +1,21 @@
+AAA diameter session metrics
+================================
+
+ergw_aaa exposes prometheus gauge metric for the states of the different diameter handlers 
```

Comment:
there is a trailing whitespace in that line (says git diff)

Reply:
fixed